MonoGame.Framework/Platform/Graphics/Texture2D.DirectX.cs

```diff
@@ -28,7 +28,7 @@ public partial class Texture2D : Texture
 
         private bool _shared;
         private bool _mipmap;
-        protected SampleDescription _sampleDescription;
+        private SampleDescription _sampleDescription;
```

Comment:
make _sampleDescription private. This field was shadowed by RenderTarget2D..DirectX.cs